src/test/java/org/example/TimeZoneConverterTest.java

```diff
@@ -0,0 +1,39 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TimeZoneConverterTest {
+    @Test
+    public void testConvertInstantToNewyork(){
+        Instant instant =Instant.now();
+        LocalDateTime expected = instant.atZone(ZoneId.of("America/New_York")).toLocalDateTime();
```

Comment:
Use constant data in your tests. Like `LocalDateTime.parse("2024-08-08T12:15:00")`